contracts/masset/MassetManager.sol

```diff
@@ -585,4 +358,9 @@ contract MassetManager is
     function getProxyImplementation() external view returns (address) {
         return ERC1967UpgradeUpgradeable._getImplementation();
     }
+
+    function setMassetTokenIntermediary(address _newMAssetTokenIntermediary) external onlyOwner {
```

Comment:
natspec missing

Reply:
updated in this [commit](https://github.com/DistributedCollective/mynt/pull/17/commits/57307ba37bc62d3f003c1b59b86350579a01d092)